src/modules/Elsa.Workflows.Core/Enums/StorageDriverTag.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Elsa.Workflows;
+
+public enum StorageDriverTag
```

Comment:
Using a closed enum limits the system from being extended in the future. If we're going to introduce tagging, let's keep it open-ended and instead use strings or (long) integers.